INFO     root:test_04_user.py:82 ==========开始执行用例：user_add_03==========
INFO     root:runcase.py:31 -·-·-·-·-·-·-·-·-·-发送请求并接受信息 START-·-·-·-·-·-·-·-·-·-
INFO     root:handleyaml.py:94 Yaml测试用例基础架构检查通过
INFO     root:runcase.py:50 重组后的用例信息：{'name': '新增用户-所属用户组不存在', 'description': '在自动化测试组下新增用户，所属用户组不存在', 'request': {'data': {'userName': 'apiuser02', 'authentication': 0, 'displayName': '自动化测试用户02', 'groupId': 'test', 'roleId': 0, 'employeeId': '', 'phoneNumber': '', 'mobileNumber': '', 'email': '', 'gender': '男', 'ipAddress': '', 'leaderUserSid': '', 'locationCode': '', 'idCard': '', 'platformCode': '', 'remark': ''}, 'method': 'POST', 'address': '/api/user/v1/user', 'headers': {'Content-Type': 'application/json', 'token': 'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzUxMiJ9.eyJhdWQiOiJhcHBsaWNhdGlvbiIsInN1YiI6IjJhZGEyNThhLTFlYzItNGM4MC1iMDkwLTY4ZWRiNTM5NzY4NyIsImlzcyI6ImJtc19zZXJ2ZXIiLCJzZXNzaW9uSWQiOiIyYWRhMjU4YS0xZWMyLTRjODAtYjA5MC02OGVkYjUzOTc2ODciLCJleHAiOjE2ODk1MDU4NzUsImlhdCI6MTY4OTMyNTg3NX0.snUHxq4voLOkjJGYIcaMc_bLfOB0Gkf83gUiIF2T5oUEqU9uDjS44RrclTSHVLocX7sFjTY8LJvd35FcSxt3IQ', 'Referer': 'https://192.168.148.174:31000/sub-app-unity/group'}}, 'validate': {'code': 400}, 'epic': '用户管理', 'feature': '基础数据管理-用户与机构管理', 'story': '新增用户接口', 'base_url': 'https://192.168.148.174:31000', 'caseid': 'user_add_03'}
INFO     root:runcase.py:81 请求地址：https://192.168.148.174:31000/api/user/v1/user
INFO     root:runcase.py:82 请求方法：POST
INFO     root:runcase.py:83 请求头：{'Content-Type': 'application/json', 'token': 'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzUxMiJ9.eyJhdWQiOiJhcHBsaWNhdGlvbiIsInN1YiI6IjJhZGEyNThhLTFlYzItNGM4MC1iMDkwLTY4ZWRiNTM5NzY4NyIsImlzcyI6ImJtc19zZXJ2ZXIiLCJzZXNzaW9uSWQiOiIyYWRhMjU4YS0xZWMyLTRjODAtYjA5MC02OGVkYjUzOTc2ODciLCJleHAiOjE2ODk1MDU4NzUsImlhdCI6MTY4OTMyNTg3NX0.snUHxq4voLOkjJGYIcaMc_bLfOB0Gkf83gUiIF2T5oUEqU9uDjS44RrclTSHVLocX7sFjTY8LJvd35FcSxt3IQ', 'Referer': 'https://192.168.148.174:31000/sub-app-unity/group'}
INFO     root:runcase.py:93 请求参数：{'userName': 'apiuser02', 'authentication': 0, 'displayName': '自动化测试用户02', 'groupId': 'test', 'roleId': 0, 'employeeId': '', 'phoneNumber': '', 'mobileNumber': '', 'email': '', 'gender': '男', 'ipAddress': '', 'leaderUserSid': '', 'locationCode': '', 'idCard': '', 'platformCode': '', 'remark': ''}
INFO     root:runcase.py:155 请求接口结果： {'timestamp': '2023-07-14T09:15:08.104+00:00', 'status': 400, 'error': 'Bad Request', 'path': '/user'}
INFO     root:checkresult.py:152 code断言通过, 期望状态码:400, 实际状态码:400
INFO     root:runcase.py:61 -·-·-·-·-·-·-·-·-·-发送请求并接受信息 END-·-·-·-·-·-·-·-·-·-